src/convertToObject.js

```diff
@@ -13,6 +13,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const styles1 = sourceString.split(';');
+
+  const styles2 = styles1.map((elem) => elem.split(':'));
+
+  const styles3 = styles2.map((elem) => elem.map((item) =>
+    item.replace(/\s+/g, ' ')));
+
+  const styles4 = styles3.filter(elem => elem.length > 1);
+
+  const styles5 = styles4.map(elem => elem.map((item) =>
+    item.trim()));
+
+  const res = {};
```

Comment:
```suggestion
  const result = {};
```
Only 3 more letters but much more sense